<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Web_page</class>
 <widget class="QDialog" name="Web_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="reference_style_box">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>274</y>
     <width>82</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLineEdit" name="author_last_bar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="translator_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>168</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Translator</string>
   </property>
  </widget>
  <widget class="QLabel" name="author_first">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>First</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="translator_multi">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>mutiple-translator</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="editor_multi">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>mutiple-editor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editor_last_bar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="editor_label">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>132</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Editor</string>
   </property>
  </widget>
  <widget class="QLabel" name="author_label">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>94</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="author_checkBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>mutiple-author</string>
   </property>
  </widget>
  <widget class="QLabel" name="translator_first">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>190</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>First</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="translator_last_bar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="editor_first">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>First</string>
   </property>
  </widget>
  <widget class="QLabel" name="contributor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>93</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Contributor</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="editor_check">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Editor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="translator_first_bar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>169</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editor_first_bar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="author_first_bar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="author_last">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QLabel" name="editor_last">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QLabel" name="translator_last">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="translator_check">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Translator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_citation_Button">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>300</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Get Citation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_Button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>74</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="citationInPaperCLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>In-text Citation</string>
   </property>
   <property name="buddy">
    <cstring>citationInPaperCLineEdit</cstring>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="5,0">
    <item row="0" column="1">
     <widget class="QPushButton" name="back_Button">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Web&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="websiteTitleWLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>212</y>
     <width>86</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Website Title(&amp;W)</string>
   </property>
   <property name="buddy">
    <cstring>website_title_line</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="uRLOfWebsiteULabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>242</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>URL of Website(&amp;U)</string>
   </property>
   <property name="buddy">
    <cstring>url_line</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="url_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="website_title_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="articleTitleALabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>63</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Article Title(&amp;A)</string>
   </property>
   <property name="buddy">
    <cstring>article_title_line</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="article_title_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>98</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Date Published</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="Date_Publish_Month_Box">
      <item>
       <property name="text">
        <string>January</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>February</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>March</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>April</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>May</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>June</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>July</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>August</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>September</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>October</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>November</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>December</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="year_label">
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="date_line"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="citationInPaperCLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="bibliographyBLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reference</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="English_checkBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>61</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="bibliographyBLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>341</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>article_title_line</tabstop>
  <tabstop>author_first_bar</tabstop>
  <tabstop>author_last_bar</tabstop>
  <tabstop>editor_first_bar</tabstop>
  <tabstop>editor_last_bar</tabstop>
  <tabstop>translator_first_bar</tabstop>
  <tabstop>translator_last_bar</tabstop>
  <tabstop>website_title_line</tabstop>
  <tabstop>url_line</tabstop>
  <tabstop>date_line</tabstop>
  <tabstop>bibliographyBLineEdit</tabstop>
  <tabstop>citationInPaperCLineEdit</tabstop>
  <tabstop>get_citation_Button</tabstop>
  <tabstop>clear_Button</tabstop>
  <tabstop>author_checkBox</tabstop>
  <tabstop>editor_check</tabstop>
  <tabstop>reference_style_box</tabstop>
  <tabstop>translator_check</tabstop>
  <tabstop>English_checkBox</tabstop>
  <tabstop>translator_multi</tabstop>
  <tabstop>back_Button</tabstop>
  <tabstop>Date_Publish_Month_Box</tabstop>
  <tabstop>editor_multi</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
